### PR TITLE
Expand pagination test

### PR DESCRIPTION
**Pull request by lbam**
Now covers JSON and tests page=2 as well. Serves as a regression test for eef66e13e7beef6c841ed09e7ab75bc656347d34.

### CODE REVIEW

File: src/tests/test_dynamic_api/test_views_api.py [hunk 1]
```diff
@@ -1804,67 +1805,147 @@ def test_unpaginated_list(self, format, api_client, afval_container, filled_rout
...
 
     @pytest.mark.parametrize("format", sorted(PAGINATED_FORMATS.keys()))
+    @pytest.mark.parametrize("page_num", (1, 2))
     @pytest.mark.parametrize("page_size_param", ["_pageSize", "page_size"])
     def test_paginated_list(
-        self, format, page_size_param, api_client, afval_container, filled_router
+        self, format, page_num, page_size_param, api_client, afval_container, filled_router
     ):
         """Prove that the pagination still works if explicitly requested."""
         decoder, expected_type, expected_data = self.PAGINATED_FORMATS[format]
```
Comment (jjmurre) [inline src/tests/test_dynamic_api/test_views_api.py:1951]: Maybe give `expected_data` a better name, so that it is clearer that it is a callable.

**Review by jjmurre — APPROVED** [review]
LGTM, on request one variable naming.

File: src/tests/test_dynamic_api/test_views_api.py (hunk 1 above)
Comment (lbam) [reply src/tests/test_dynamic_api/test_views_api.py]: Renamed to `make_expected`, please re-review.

**Review by jjmurre — APPROVED** [review]
LGTM

**Pull request merged**